docs/docsite/rst/playbooks_intro.rst

```diff
@@ -368,15 +360,14 @@ Action Shorthand
 
 .. versionadded:: 0.8
 
-Ansible prefers listing modules like this in 0.8 and later::
+Ansible prefers listing modules like this::
```

Comment:
Since we're telling people a best practice here, we should probably change the example to this:
```
template:
    src: templates/foo.j2
    dest: /etc/foo.conf
```

Reply:
Done. Thanks.